x-pack/solutions/security/plugins/security_solution/public/common/components/charts/barchart.tsx

```diff
@@ -192,8 +191,8 @@ export const BarChartComponent: React.FC<BarChartComponentProps> = ({
   const legendItems: LegendItem[] = useMemo(
     () =>
       barChart != null && stackByField != null
-        ? barChart.map((d, i) => ({
-            color: d.color ?? (i < defaultLegendColors.length ? defaultLegendColors[i] : undefined),
```

Comment:
`defaultLegendColors` is removed as we don't want hard coded colors:
x-pack/solutions/security/plugins/security_solution/public/common/components/matrix_histogram/utils.ts